src/hikogui/delayed_format.hpp

```diff
@@ -66,13 +66,13 @@ class delayed_format {
     template<typename... Args>
     static std::string format_wrapper(Args const &...args)
     {
-        return std::format(static_cast<std::string_view>(Fmt), args...);
+        return std::vformat(static_cast<std::string_view>(Fmt), std::make_format_args(args...));
```

Comment:
I need to look at this, because delayed_format uses a compile time constant format-string and it should be possible to forward that to std::format.